### PR TITLE
WF-65704-Added ASP.NET Core and Windows Forms for open and save Word document

### PR DESCRIPTION
**Pull request by elizabethatienoo**
<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-65704">https://syncfusion.atlassian.net/browse/WF-65704</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Added UG documentation for ASP.NET Core and Windows Forms to open and save Word document
</td>
</tr>
<tr>
<td>
Solution
</td>
<td>
Added UG documentation for ASP.NET Core and Windows Forms to open and save Word document
</td>
</tr>
<tr>
<td>
Content review confirmation
</td>
<td>
 Pending
</td>
</tr>
</table>

### CODE REVIEW

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-Core.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+{% endtabs %}
+
+Step 5: A default action method named Index will be present in HomeController.cs. Right click on Index method and select **Go To View** where you will be directed to its associated view page **Index.cshtml**.
+
+Step 6: Add a new button in the Index.cshtml as shown below.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-Core.md:42]: {% highlight HTML %}

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-Core.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+title: Open and save Word document in ASP.NET Core | Syncfusion
+description: Open and save Word document in ASP.NET Core application using Syncfusion .NET Core Word (Essential DocIO) library without Microsoft Word or interop dependencies
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Open and save Word document in ASP.NET Core
+
+Syncfusion Essential DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net-core/word-library) used to create, read, and edit **Word** documents programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **open and save a Word document in ASP.NET Core**.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-Core.md]: Check and modify the sentence as per existing feedbacks

File: File-Formats/DocIO/Open-and-save-Word-document-in-Windows-Forms.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+Step 5: Add the following code in **btnOpenAndSave_Click** to **Open and save Word document** with simple text.
+
+**Open an existing Word document:**
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Open an existing Word document.
+using (WordDocument document = new WordDocument(Path.GetFullPath(@"../../Data/Input.docx"), FormatType.Automatic))
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-Windows-Forms.md:78]: using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))

File: File-Formats/DocIO/Open-and-save-Word-document-in-Windows-Forms.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+{
+
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+**Add a paragraph to the Word document:**
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-Windows-Forms.md:86]: Refer this file and change the steps and remove this line
https://github.com/syncfusion-content/fileformat-docs/blob/hotfix/hotfix-v21.1.35/File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET.md

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats-toc.html [hunk 5]
```diff
@@ -120,6 +120,11 @@
             </li>
             <li>
                 <a href="/file-formats/DocIO/Loading-and-Saving-document">Loading and Saving document</a>
+				<ul>
```
Comment (MohanaselvamJothi) [inline File-Formats-toc.html:123]: Rebase the branch properly to hotfix, some changes are missing

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-Core.md (hunk 2 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-Core.md]: Modified

File: File-Formats/DocIO/Open-and-save-Word-document-in-Windows-Forms.md (hunk 3 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Open-and-save-Word-document-in-Windows-Forms.md]: Modified

File: File-Formats/DocIO/Open-and-save-Word-document-in-Windows-Forms.md (hunk 4 above)
Comment (elizabethatienoo) [reply File-Formats/DocIO/Open-and-save-Word-document-in-Windows-Forms.md]: Done

File: File-Formats-toc.html [hunk 6]
```diff
@@ -121,9 +121,11 @@
             <li>
                 <a href="/file-formats/DocIO/Loading-and-Saving-document">Loading and Saving document</a>
 				<ul>
+					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-ASP-NET-Core">ASP.NET Core</a></li>
 					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC">ASP.NET MVC</a></li>
 					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-ASP-NET">ASP.NET</a></li>
 					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-WPF">WPF</a></li>
+					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-Windows-Forms">Windows Forms</a></li>
```
Comment (MohanaselvamJothi) [inline File-Formats-toc.html:128]: Can you please change the order of WPF and Windows Forms. Refer getting started page

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Pull request merged**